Silicon/ElkhartlakePkg/Include/Register/PchRegsSpi.h

```diff
@@ -42,6 +42,9 @@
 
 #define R_SPI_CFG_BAR0                      0x10
 
+#define R_SPI_CFG_BC                        0xDC
+#define B_SPI_CFG_BC_EISS                   BIT5            ///< Enable InSMM.STS
+#define B_SPI_CFG_BC_LE                     BIT1            ///< Lock Enable

```

Comment:
Can we use the ones from CommonSocPkg?

Reply:
I've changed it to use the register from CommonSocPkg, thanks @myguitar.